tensorflow/training/docker/2.0.2/py3/cu100/Dockerfile.gpu

```diff
@@ -176,7 +176,7 @@ RUN ${PIP} install --no-cache-dir -U \
     sagemaker-experiments==0.* \
     "sagemaker-tensorflow>=2.0,<2.1" \
     "sagemaker-tensorflow-training>=3.0<4.0" \
-    smdebug==0.8.1 \
+    smdebug==0.9.2 \
```

Comment:
Please make this an ARG like the other dockerfiles

Reply:
Done.